docs/desktop-wpf/fundamentals/styles-templates-create-apply-template.md

```diff
@@ -0,0 +1,183 @@
+
+In most user interfaces, a button has the same general appearance: a rectangle with some text. If you wanted to create a rounded button, you normally have to create a new control that inherits from the button or recreates the functionality of the button. Your new user control would then provide that circular visual.
+
+You can avoid creating new controls by customizing the visual layout of an existing control. With a rounded button, you create a <xref:System.Windows.Controls.ControlTemplate> with the wanted visual layout.
+
+On the other hand, if you need a control with new functionality, different properties, and new settings, you would create a new <xref:System.Windows.Controls.UserControl>.
+
+## Prerequisites
+
+Create a new WPF application and in *MainWindow.xaml* (or another window of your choice) set the following properties on the `<Window>` element:
+
+|     |     |
+| --- | --- |
+| **Title**         | `Template Intro Sample` |
+| **SizetoContent** | `WidthAndHeight` |
+| **MinWidth**      | `250` |
+
+Set the content of the `<Window>` element to the following XAML:
+
+[!code-xaml[Initial](~/samples/snippets/desktop-guide/wpf/styles-templates-create-apply-template/csharp/Window1.xaml#Initial)]
+
+In the end, the *MainWindow.xaml* file should look similar to the following:
+
+[!code-xaml[InitialWhole](~/samples/snippets/desktop-guide/wpf/styles-templates-create-apply-template/csharp/Window1.xaml#InitialWhole)]
+
+If you run the application, it looks like the following:
+
+![WPF window with two unstyled buttons](media/create-apply-template/unstyled-button.png)
+
+## Create a ControlTemplate
+
+The most common way to declare a <xref:System.Windows.Controls.ControlTemplate> is as a resource in the `Resources` section in a XAML file. Because templates are resources, they obey the same scoping rules that apply to all resources. Put simply, where you declare a template affects where the template can be applied. For example, if you declare the template in the root element of your application definition XAML file, the template can be used anywhere in your application. If you define the template in a window, only the controls in that window can use the template.
+
+To start with, add a `Window.Resources` element to your *MainWindow.xaml* file:
+
+[!code-xaml[WindowResStart](~/samples/snippets/desktop-guide/wpf/styles-templates-create-apply-template/csharp/Window2.xaml#WindowResStart)]
+
+Create a new `<ControlTemplate>` with the following properties set:
+
+|     |     |
+| --- | --- |
+| **x:Key**         | `roundbutton` |
```

Comment:
Why not pivot the table to have Key and TargetType as column headers, so they'll be automatically bolded?

Reply:
I just think of it as a list, like the table above this one with 3 properties. If you have a big set of property changes, then you can't pivot otherwise you have a long horizontal line of settings.